src/main/kotlin/racing/domain/Cars.kt

```diff
@@ -0,0 +1,26 @@
+package racing.domain
+
+private const val MINIMUM_COUNT_OF_CARS = 2
+
+class Cars(private val cars: List<Car>) {
+
+    init {
+        require(cars.size >= MINIMUM_COUNT_OF_CARS) {
+            "자동차는 최소 ${MINIMUM_COUNT_OF_CARS}대 이상이어야 합니다."
+        }
+    }
+
+    constructor(numberOfCars: Int) : this(
+        (1..numberOfCars).map { Car() }
```

Comment:
아래와 같이 변경해볼 수 있을 것 같아요. 😉 
```suggestion
        List(numberOfCars) { Car() }
```

Reply:
👍 👍